core/typescript/package.json

```diff
@@ -45,7 +45,7 @@
     "cov:html": "nyc report --reporter=html",
     "cov:lcov": "nyc report --reporter=lcov",
     "cov:send": "run-s cov:lcov && codecov",
-    "cov:check": "nyc report && nyc check-coverage --lines 80 --functions 80 --branches 80",
+    "cov:check": "nyc report && nyc check-coverage --lines 100 --functions 100 --branches 100",
```

Comment:
Dialling it up to 100% as I find *cov* very helpful in discovering untested branches. My old thinking of 80% is good enough doesn't seem to apply here as *cov* has its own metrics that seem to suggest 100% is attainable and should be

Reply:
yeah, execution branch coverage used to be the key diff between CodeCov and the rest. I think most have caught up, maybe the expensive Enterprise offerings  has it anyways. I think https://coveralls.io/ now also support execution branch coverage and multi lang mono repos.